### PR TITLE
Vlad

### PR DESCRIPTION
**Pull request by vm0ise**
Added Car and CarType tables in models, updated URLS, Views, Serializers.

### CODE REVIEW

File: car_rental/api/models.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
+from email.policy import default
```
Comment (axelnguyen0701) [inline car_rental/api/models.py:1]: Is this needed?

File: car_rental/api/models.py [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+from pyexpat import model
```
Comment (axelnguyen0701) [inline car_rental/api/models.py:2]: Is this needed?

File: car_rental/api/models.py [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+from random import choices
```
Comment (axelnguyen0701) [inline car_rental/api/models.py:3]: Is this needed?

File: car_rental/api/models.py [hunk 4]
```diff
@@ -1,4 +1,8 @@
...
 from django.db import models
+from django.forms import FloatField
```
Comment (axelnguyen0701) [inline car_rental/api/models.py:5]: FloatField is supported automatically in djangoDB right?

File: car_rental/api/serializers.py [hunk 5]
```diff
@@ -1,7 +1,18 @@
+from cgi import MiniFieldStorage
```
Comment (axelnguyen0701) [inline car_rental/api/serializers.py:1]: Why is this needed?

**Review by axelnguyen0701 — CHANGES_REQUESTED** [review]
Some import statement seems unnecessary.

File: car_rental/api/models.py (hunk 4 above)
Comment (vm0ise) [reply car_rental/api/models.py:5]: yeah, it is...when I first used it, VS code automatically imported it for me for some reason.

Comment (axelnguyen0701) [reply car_rental/api/models.py:5]: yeah could you try removing it and test if your code still works without this import? Then push your work

File: car_rental/api/models.py (hunk 3 above)
Comment (vm0ise) [reply car_rental/api/models.py:3]: no I don't think they are...I'm not sure what the reason for those imports for VS Code were.

**Comment by vm0ise** [comment]
making changes to models
